internal/oci/uvm.go

```diff
@@ -352,9 +352,21 @@ func SpecToUVMCreateOpts(ctx context.Context, s *specs.Spec, id, owner string) (
 // UpdateSpecFromOptions sets extra annotations on the OCI spec based on the
 // `opts` struct.
 func UpdateSpecFromOptions(s specs.Spec, opts *runhcsopts.Options) specs.Spec {
```

Comment:
Oh this is odd. Its passed by value and returned by value. Do we need this? Seems wasteful